webdriver/tests/bidi/browser/create_user_context/proxy.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from .. import get_user_context_ids
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.fixture
+def create_user_context_with_proxy_and_assert_it_is_created(bidi_session,
```

Comment:
I'm not sure if you have already an idea how to test all the proxy types. I personally only have an idea about manual type and maybe direct (but only in combination with global proxy set to manual). So maybe it makes sense to keep this fixture as to assert that a user context is created and have a custom logic to verify that proxy applies in each case?